stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -256,7 +256,7 @@ public struct _ContiguousArrayBuffer<Element> : _ArrayBufferProtocol {
 
   /// If the elements are stored contiguously, a pointer to the first
   /// element. Otherwise, `nil`.
-  public var firstElementAddress: UnsafeMutablePointer<Element> {
+  public var firstElementAddress: UnsafeMutablePointer<Element>? {
```

Comment:
Unfortunately a non-Optional property will not satisfy an Optional protocol requirement.